roles/common/defaults/main.yml

```diff
@@ -110,6 +110,7 @@ prereq_option: "{% if oracle_ver_base in ('11.2', '12.1') %}-ignorePrereq{% else
 # (like "_sydney" for example)
 # The same suffix will be "attached" to the db name in DG configuration as well.
 standby_suffix: _s
```

Comment:
If standby_suffix isn't otherwise used, how about just setting standby_name "{{ db_name }}_s"

It brings up a bigger question though:  if there's a role switch, now _s isn't the standby anymore.  It's the primary.  Not sure if there's an easy way to generalize here;  Orafcle's examples typically use geographic locations, but standbys aren't always in a different place.